tests/vhost/tests.c

```diff
@@ -82,6 +82,7 @@ int main(int argc, char **argv)
 	/* initialize packetgraph */
 	ret = pg_start(argc, argv);
 	g_assert(ret >= 0);
+//	g_assert(!pg_init_seccomp());
```

Comment:
:( 
you can remove that line 
I'm a little sad, because without this line, this PR would be merge